test_draft.py

```diff
@@ -8,7 +8,7 @@
 from inference.probability import JointDistribution
 from player import ProbabilisticAIPlayer
 from utils import Point
-from wumpus import create_wumpus_world, create_wumpus_world2
+from wumpus import create_wumpus_world1, create_wumpus_world2
```

Comment:
You need to add the new class import as well:

```suggestion
from wumpus import create_wumpus_world1, create_wumpus_world2, WumpusWorldGenerator
```